### DISCUSSION TITLE
Permit third parties to submit PR's with documentation update

### DISCUSSION THREAD

**Discussion by noahisapilot** [romeerez/orchid-orm#54] [Ideas]
I have an update to the docs but I don't see a way to open a PR against them. I noticed the docs are published on netlify but I don't see a repo for this. I think it would be useful to allow people to publish updates to docs.

In my particular case, the update relates to the [docs around json types](https://orchid-orm.netlify.app/guide/columns-json-types.html). The following is the first example code and all the examples follow the same pattern:

```typescript
    data: t.json((t) => ({
      age: t.number(),
      name: t.string(),
      description: t.string().optional(),
      tags: t.string().array(),
    })),
```

but this actually does not pass type check. Instead, this should be:

```typescript
    data: t.json((j) => j.object({
      age: j.number(),
      name: j.string(),
      description: j.string().optional(),
      tags: j.string().array(),
    })),
```

(note the `j.object`)

**Comment by romeerez** [comment]
Docs are in this repo in packages/docs, thanks for pointing - I should write somewhere about it so people can find it.

All packages are using `pnpm`, but not the docs, docs are on regular `npm` because it's easier for netlify. It would make sense to move the docs from `packages` to the top level, then it would be easier to find and less confusing as it's not a regular package.
